### PR TITLE
add upgrade file with information how to upgrade extensions

### LINKED ISSUE #212: Upgrade File

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
fix #212

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
@dbu @wouterj error can't be related to my little change, do we have that one on master too?

**Comment by wouterj** [comment]
Yes, you can ignore it. Other PR is fixing it.

File: UPGRADE-2.0.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+UPGRADE FROM 1.x to 2.0
+=======================
+
+### SonataAdmin Support
+
+ * The Admin extension to edit seo metadata was moved into `symfony-cmf/sonata-admin-integration-bundle`.
```
Comment (wouterj) [inline UPGRADE-2.0.md]: This sentence needs to be updated (copy paste error from seo)

File: UPGRADE-2.0.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+   With the move, the admin extension service names also changed. If you are using the one of the core extensions,
```
Comment (wouterj) [inline UPGRADE-2.0.md]: Remove "the" from "using the one of ..."

File: UPGRADE-2.0.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    After:
+       
+   ```yaml
+        # app/config/config.yml
+                
+        sonata_admin:
+            extensions:
+                 cmf_sonata_admin_integration.core.extension.child:
+                    implements:
+                        Symfony\Cmf\Bundle\CoreBundle\Model\ChildInterface
```
Comment (wouterj) [inline UPGRADE-2.0.md]: Missing - prefix?

**Pull request merged**